src/components/NowPlaying.jsx

```diff
@@ -1,26 +1,113 @@
-import SwitchIcon from "../assets/icons/switch.svg?react"
-import PreviousIcon from "../assets/icons/previous.svg?react"
-import PauseIcon from "../assets/icons/pause.svg?react"
-import NextIcon from "../assets/icons/next.svg?react"
-import DisablerepeatIcon from "../assets/icons/disablerepeat.svg?react"
-import NowplayingviewIcon from "../assets/icons/nowplayingview.svg?react"
-import QueueIcon from "../assets/icons/queue.svg?react"
-import ConnecttodeviceIcon from "../assets/icons/connecttodevice.svg?react"
-import MuteIcon from "../assets/icons/mute.svg?react"
-
+import SwitchIcon from '../assets/icons/switch.svg'
+import PreviousIcon from '../assets/icons/previous.svg'
+import PauseIcon from '../assets/icons/pause.svg'
+import PlayIcon from '../assets/icons/play.svg'
+import NextIcon from '../assets/icons/next.svg'
+import DisableRepeatIcon from '../assets/icons/disableRepeat.svg'
+import NowplayingviewIcon from '../assets/icons/nowplayingview.svg'
+import QueueIcon from '../assets/icons/queue.svg'
+import ConnecttodeviceIcon from '../assets/icons/connecttodevice.svg'
+import MuteIcon from '../assets/icons/mute.svg'
+import TrackIcon from '../assets/icons/track.svg'
+import { useGetEntity } from '../hooks/useGetEntity.jsx'
+import { useEffect, useRef, useState } from 'react'
+import YouTube from 'react-youtube'
+import style from '../assets/styles/modules/youtube.module.scss'
 
 export const NowPlaying = () => {
+    const { track, /*error, status*/ } = useGetEntity('track', 'skyfall')
+    console.log(track)
+
+    // Function to construct the YouTube URL
```

Comment:
Please delete this comment. Perhaps, construct URL logic should be moved to a file intended for Youtube service, what do you say? @moria0525

Reply:
I agree